report.py

```diff
@@ -48,6 +50,35 @@ def get_translations(cls):
         return JinjaTranslator(lambda text: translate(text))
 
 
+class Jinja2ReportLogo(Jinja2Report):
+
+    @classmethod
+    def render(cls, action, context):
+        context['logo'] = cls.get_logo(context['user'])
```

Comment:
I think it will be better to put a `get_attachment` utility function in the context, that would receive an arbitrary record and an arbitrary filename from the template and return the raw byte content of the file.

This way, potentially any attachment for any record could be used in any template, and let template designers decide where to get the file from. for example: the context user's company will be unsuitable if an user with access to multiple companies prints an invoice from a different company context. Or a template designer may have access to different svg and jpg and tiff attachments and would like to use them all in the same template.

I think this development is being tailored for a very specific need when a more general implementation will be easier and deserve a better place in a public library!